docs/docsite/rst/reference_appendices/YAMLSyntax.rst

```diff
@@ -79,6 +79,8 @@ Ansible doesn't really use these too much, but you can also specify a boolean va
     knows_oop: True
     likes_emacs: TRUE
     uses_cvs: false
+    
+If you care about code-quality by using `yamllint`, or `molecule`, then you will use the lowercase `true` and `false`.
```

Comment:
Recommend re-phrasing this to something like "Use lowercase 'true' or 'false' for boolean values in dictionaries to ensure good code quality."

Reply:
Amended.

Reply:
Looks good, thanks @bbaassssiiee